flink-addons/flink-language-binding/src/main/python/org/apache/flink/languagebinding/api/python/dill/__diff.py

```diff
@@ -0,0 +1,247 @@
+################################################################################
```

Comment:
can't we just remove the apache license from these files, and throw into the notice file or wherever that file x,y is licensed under license z?